Assignment2/JavierTrigosDominguez.ttl

```diff
@@ -0,0 +1,16 @@
+@base <http://example.org/>.
+@prefix : <http://example.org/elements>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema>.
+
+:Class01 :includes :Sensor029;
+         :includes :Computer101.
+		 
+:Computer101 :hasOwner :User10A.
+
+:User10A :hasName "Pedro".
+		 
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Measurement8401 :hasTemperature "29";

```

Comment:
Check other accepted files, to see how you have to provide data types for the integer and dateTime literals